doc/source/reference/routines.testing.rst

```diff
@@ -6,8 +6,9 @@ Test Support (:mod:`numpy.testing`)
 Common test support for all numpy test scripts.
 
 This single module should provide all the common functionality for numpy
-tests in a single location, so that test scripts can just import it and
-work right away.
+tests in a single location, so that :ref:`test scripts
+<development-environment>` can just import it and work right away. For
+background, see the :ref:`Testing Guidelines <testing-guidelines>`
```

Comment:
changed to use the title from the link.

This is meant to link to `doc/source/reference/testing.rst`, which includes the file in `doc/TESTS.rst.txt` and is turned into `<html_root>/reference/testing.html`, I used the `include` trick to prevent having to move the testing guidelines doc, since downstream packages [reference it](https://docs.scipy.org/doc/scipy-0.18.1/reference/hacking.html#contributing-new-code) directly from the github source link.

I tested it by `git clean -xfd; pip install .; cd doc; make html`. It works locally.